src/components/Footer.tsx

```diff
@@ -34,7 +34,11 @@ const Footer = () => {
         <div className={styles.iconContainer}>
           {footerIcons.map((icon) => {
             return (
-              <a href={`https://www.${icon.iconName}.com`} target="_blank">
+              <a
+                href={`https://www.${icon.iconName}.com`}
```

Comment:
it seems strange/unrealistic to have only the 2nd level domain section of the url (iconName) a variable, not the protocol (https), subdomain (www), or top-level domain (.com).  I think it would probably make more sense to have the fully qualified domain in a constants file and import it from there...